core/arch/arm/plat-versal/platform_config.h

```diff
@@ -0,0 +1,69 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2022 - All Rights Reserved
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+
+#ifdef CFG_WITH_PAGER
+#error "Pager not supported for Versal"
+#endif
```

Comment:
If "not supported" is only due to not having any boot loader config for that purpose, I'd rather avoid that error. It could be useful to accept `CFG_WITH_PAGER=y` if only for build testing.

Reply:
ok will do that but currently it just wont boot (I havent looked into it TBH)